cypress/e2e/po/edit/provisioning.cattle.io.cluster/create/cluster-create-rke1-custom.po.ts

```diff
@@ -3,16 +3,17 @@ import ClusterManagerCreatePagePo from '@/cypress/e2e/po/edit/provisioning.cattl
 import EmberInputPo from '@/cypress/e2e/po/components/ember/ember-input.po';
 import ClusterManagerCreateRKE1PagePo from '@/cypress/e2e/po/edit/provisioning.cattle.io.cluster/create/cluster-create-rke1.po';
 import EmberAccordionPo from '@/cypress/e2e/po/components/ember/ember-accordion.po';
-import EmberFormMembersPo from '~/cypress/e2e/po/components/ember/ember-form-members.po';
+import EmberFormMembersPo from '@/cypress/e2e/po/components/ember/ember-form-members.po';
+import EmberAgentConfigurationPo from '@/cypress/e2e/po/components/ember/ember-agent-configuration.po';
 
 /**
  * Create page for an RKE1 custom cluster
  */
 export default class ClusterManagerCreateRke1CustomPagePo extends ClusterManagerCreateRKE1PagePo {
   static url = `${ ClusterManagerCreatePagePo.url }/create?type=custom`
-  static goTo(): Cypress.Chainable<Cypress.AUTWindow> {
-    return PagePo.goTo(ClusterManagerCreateRke1CustomPagePo.url);
-  }
+  // static goTo(): Cypress.Chainable<Cypress.AUTWindow> {
```

Comment:
How come this was commented out?

Reply:
Something I was testing while I was trying to work out how to make the edit page po and forgot to revert - un-commented now